Helper/Paymentorder.php

```diff
@@ -437,6 +437,7 @@ public function saveQuoteToDB($response)
         }
 
         $quote->setPaymentOrderId($this->getSwedbankPayPaymentorderId($response['payment_order']['id']));
+        $quote->setPaymentIdPath($response['payment_order']['id']);
```

Comment:
This is weird. Here you're passing `paymentOrder.id` to a method called `setPaymentIdPath`?

```suggestion
        $quote->setPaymentOrderId($response['payment_order']['id']);
```

Reply:
Since there's already a method called `setPaymentOrderId()` that stores the last part of the ID, we have created another method called `setPaymentIdPath()` to store the entire ID & have backwards compatibility.

Reply:
Can't `setPaymentOrderId` be changed to take the entire URI (path)? Doing string split on the URI to extract the GUID is a huge error.

Reply:
Yes, we are now setting the entire URI/Path